### PR TITLE
add softlayer getting started

### PR DESCRIPTION
**Pull request by andreaturli**

### CODE REVIEW

**Comment by andreaturli** [comment]
thanks @andrewgaul, I've removed the reference to the inputStream.

**Comment by jclouds-commentator** [comment]
Go to http://38c6097802420a7a2cc0-4b157843f53d27e5c682ab34efd3c4b1.r81.cf5.rackcdn.com/ to review your changes.

File: guides/index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Use these guides for getting started with any of the cloud providers below:
 * [OpenStack](openstack)
 * [ProfitBricks](profitbricks)
 * [Rackspace](rackspace)
+* [SoftLayer](softlayer-cci)
```
Comment (nacx) [inline guides/index.md]: Broken link?

**Comment by nacx** [comment]
apart from the broken link, lgtm. Thanks @andreaturli!

**Comment by nacx** [comment]
Fixed the broken link, merged and published.

**Comment by andreaturli** [comment]
Thanks @nacx !

**Pull request closed without merging**